2-mandatory/1-create-functions.js

```diff
@@ -24,7 +29,10 @@ Write a function that:
 - Removes any forward slashes (/) in the strings.
 - Makes the strings all lowercase.
 */
-function tidyUpString() {
+function tidyUpString(array) {
+  return array.map((str)=>{
+    return str.trim().replace("/","").toLowerCase();
+  });
 }
```

Comment:
Although the rules for this function state "Removes any forward slashes (/) in the strings.", the data in the test contains only one forward slash, so doing ".replace("/","") makes the test pass, though this form of replace, replaces just the first occurrence of forward slash. If the input string had two or more forward slashes, your code would have failed. You can globally replace a string using this form of replace:
```javascript
str.replace(/\//g, "")
```
Whilst this looks strange, it's unfortunate that regular expressions start and end with the same character (forward slash) that we want to target, so to do so, we have to prefix the forward slash character with a backslash character. This is called "escaping" a character when you prefix it with a backslash. The "g" on the end of the regular expression is a flag used to mean global, i.e. when this match is found anywhere in "str", replace the matched characters with an empty string.